spec/02-access_spec.lua

```diff
@@ -90,7 +90,7 @@ describe("Plugin: prometheus (access)", function()
         path    = "/metrics",
       })
       local body = assert.res_status(200, res)
-      return body:find('kong_http_status{code="200",service="mock-service",route="http-route"} 1', nil, true)
```

Comment:
curious, why was this changed?

Reply:
I see why now. You swapped the order of labels. Why was that necessary?
Does changing table_3[3] over table_3[1] have a performance difference in Lua/LuaJIT?

Reply:
this is to save some memory usage, since in the prometheus library we create a new table for each diverged label, since we expect more than one status code per metrics, so putting the status code at the end will save us some tables being created. I'll put this in the comment